tools/subgraph-example/subgraph/networks.json

```diff
@@ -1,16 +1,24 @@
 {
   "local": {
     "ExampleERC721": {
-      "address": "0x000000000000000000000000000000000000040A",
-      "startBlock": 116
+      "address": "0x8A7fa94487d0d0460550e5F3F80A663c39Ac8B10",
+      "startBlock": 246
```

Comment:
Q: how do we get these starting block numbers?

Reply:
Automatically generated upon deploying the contract/tokens.

Reply:
Interesting.
Is it thr same for each run or can this vary?

Reply:
It varies, depending on the run. It takes the block when it deploys the contract/token.